LayoutTests/imported/w3c/web-platform-tests/digital-credentials/disabled-by-permissions-policy.https.sub-expected.txt

```diff
@@ -0,0 +1,6 @@
+
+
+FAIL Permissions-Policy header digital-credentials-get=() disallows the top-level document. promise_rejects_dom: function "function() { throw e }" threw object "TypeError: At least one provider must be specified." that is not a DOMException NotAllowedError: property "code" is equal to undefined, expected 0
+FAIL Permissions-Policy header digital-credentials-get=() disallows same-origin iframes. assert_false: Digital Credential API expected false got true
```

Comment:
The first failure makes sense, but I'm not sure I understand this failure. Is that expected?

Reply:
I think this is correct, as we don't support the HTTP header:

* as the header has no effect, the API is behaving as `'self'`. Hence we get back true, instead of false.